src/app/routes/api/v3/forms/__tests__/public-forms.verification.routes.spec.ts

```diff
@@ -94,4 +138,82 @@ describe('public-forms.verification.routes', () => {
       })
     })
   })
+
+  describe('POST /forms/:formId/fieldverification/:transactionId/fields/:fieldId/reset', () => {
+    it('should return 200 when transactionId and fieldId for email field are valid', async () => {
```

Comment:
nit: when `formId`, transactionId and fieldId

Reply:
Will fix, thanks!